File: test/fsharpTests/IntTest/absoluteTest.fs
testCase "absolute(8)" 
<| fun _ -> 
    let expected = 8
    Expect.equal expected (Int.absolute 8) "error"
testCase "absolute(-7)" 
<| fun _ -> 
    let expected = 7
    Expect.equal expected (Int.absolute -7) "error"
testCase "absolute(0)" 
<| fun _ -> 
    let expected = 0
    Expect.equal expected (Int.absolute 0) "error"

File: test/ocamlTests/IntTest/absoluteTest.ml
test "absolute(8)" (fun () -> expect (absolute 8) |> toEqual Eq.int 8) ; 
test "absolute(-7)" (fun () -> expect (absolute -7) |> toEqual Eq.int 7) ; 
test "absolute(0)" (fun () -> expect (absolute 0) |> toEqual Eq.int 0) ; 
